assets.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
This file doesn't want to be checked in. I'm pretty sure if it doesn't exist it will be recreated.